osu.Android.props

```diff
@@ -0,0 +1,69 @@
+<Project>
+  <PropertyGroup>
+    <OutputPath>bin\$(Configuration)</OutputPath>
+    <WarningLevel>4</WarningLevel>
+    <SchemaVersion>2.0</SchemaVersion>
+    <BundleAssemblies>false</BundleAssemblies>
+    <AotAssemblies>false</AotAssemblies>
+    <LangVersion>default</LangVersion>
+    <OutputType>Library</OutputType>
+    <FileAlignment>512</FileAlignment>
+    <GenerateSerializationAssemblies>Off</GenerateSerializationAssemblies>
+    <AndroidApplication>True</AndroidApplication>
+    <AndroidHttpClientHandlerType>Xamarin.Android.Net.AndroidClientHandler</AndroidHttpClientHandlerType>
+    <TargetFrameworkVersion>v8.1</TargetFrameworkVersion>
+    <AndroidUseLatestPlatformSdk>false</AndroidUseLatestPlatformSdk>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugSymbols>True</DebugSymbols>
+    <DebugType>portable</DebugType>
+    <Optimize>False</Optimize>
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <EnableLLVM>false</EnableLLVM>
+    <AndroidManagedSymbols>false</AndroidManagedSymbols>
+    <AndroidLinkMode>SdkOnly</AndroidLinkMode>
+    <AndroidUseSharedRuntime>true</AndroidUseSharedRuntime>
+    <EmbedAssembliesIntoApk>false</EmbedAssembliesIntoApk>
+    <MandroidI18n>cjk,mideast,other,rare,west</MandroidI18n>
+    <AndroidEnableSGenConcurrent>true</AndroidEnableSGenConcurrent>
+    <AndroidSupportedAbis>armeabi-v7a;x86;arm64-v8a</AndroidSupportedAbis>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <DebugSymbols>false</DebugSymbols>
+    <DebugType>None</DebugType>
+    <Optimize>True</Optimize>
+    <ErrorReport>prompt</ErrorReport>
+	<EnableLLVM>true</EnableLLVM>
+    <AndroidManagedSymbols>false</AndroidManagedSymbols>
+    <AndroidLinkMode>SdkOnly</AndroidLinkMode>
+    <AndroidUseSharedRuntime>False</AndroidUseSharedRuntime>
+    <EmbedAssembliesIntoApk>true</EmbedAssembliesIntoApk>
+    <MandroidI18n>cjk,mideast,other,rare,west</MandroidI18n>
+    <AndroidEnableSGenConcurrent>true</AndroidEnableSGenConcurrent>
+    <AndroidSupportedAbis>armeabi-v7a;x86;arm64-v8a</AndroidSupportedAbis>
```

Comment:
Either remove the x86 ABI or add the bass android native libs for x86

Reply:
For what it's worth, I couldn't even get an ARM emulator running (not sure if it's just unsupported these days? I couldn't even use recent versions of android when creating the emulator). My minimum requirement for merging this PR is having the game run inside an emulated device on PC, so we will need to get this sorted.

Reply:
Judging from the screenshot you posted in #osu-framwork you selected a really old android image. The experience with the old ones is meh.

Also the arm ones take ages to load and even then they are really slow.

Here is a video with the x86 emulator where I added the x86 bass libraries myself:
https://drive.google.com/uc?id=1fd14UUonBJEpBWZ-nRDWg4m7albdOrSc

It is working and useable.

Reply:
Are you able to branch fro this branch and add the x86 libraries in a working state?